scripts/salt-install.sh

```diff
@@ -20,14 +20,30 @@ function install_salt_with_pip() {
   mkdir ${SALT_PATH}
   virtualenv ${SALT_PATH}
   source ${SALT_PATH}/bin/activate
-  if [ "${OS}" == "redhat7" ] ; then
+  if [ "${OS}" == "redhat7" || "${OS}" == "redhat8" ] ; then
     # can't install this via salt_requirements.txt and I dunno why...
     pip install pbr
   fi
   pip install --upgrade pip
   pip install -r /tmp/salt_requirements.txt
 }
 
+function install_salt_with_pip3() {
+
+  echo "Installing salt with version: $SALT_VERSION"
+  pip3 install --upgrade pip
```

Comment:
At some places you've removed "pip3" and replaced it with "python3 -m pip". Why do you stick to "pip3" here?